util/cookies.ts

```diff
@@ -1,37 +1,14 @@
 import cookie from 'cookie';
-import Cookies from 'js-cookie';
-
-// more robust way to get items from cookies without parsing all the time
-export function getParsedCookie(key: string): CookieValue | undefined {
-  const cookieValue = Cookies.get(key);
-
-  if (!cookieValue) {
-    return undefined;
-  }
-
-  try {
-    return JSON.parse(cookieValue); // Type should be a string
-  } catch (err) {
-    return undefined;
-  }
-}
+import { cookies } from 'next/headers';
 
 export type CookieValue = {
   id: number;
-  stars: number;
+  comment: string;
```

Comment:
this should probably be `comment?: string` for the cases where it's undefined, because the property goes away if it's undefined:

![Screenshot 2023-05-17 at 22 49 28](https://github.com/upleveled/next-js-example-winter-2023-vienna-austria/assets/1935696/727bd870-f4e8-4717-b3f0-bf9fecdc597e)

Reply:
https://github.com/upleveled/next-js-example-winter-2023-vienna-austria/blob/ae48d6bc14c2ca62a89811a454ac80f704e4b3e2/util/cookies.ts#L6